<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphWidget</class>
 <widget class="QWidget" name="graphWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>318</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>280</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Graphs</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>fvd.ico</normaloff>fvd.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="graphChooser">
       <attribute name="title">
        <string>Graph List</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="selTree">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Graph</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Show</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Editable Graphs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Resulting Graphs</string>
           </property>
           <item>
            <property name="text">
             <string>Roll</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>Banking</string>
             </property>
             <property name="background">
              <brush brushstyle="NoBrush">
               <color alpha="80">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="background">
              <brush brushstyle="NoBrush">
               <color alpha="80">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Roll Speed</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Roll Accel</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Normal Force</string>
            </property>
            <item>
             <property name="text">
              <string>Normal Force</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Force Change</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Lateral Force</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>Lateral Force</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Force Change</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Geometric</string>
            </property>
            <item>
             <property name="text">
              <string>Pitch Change</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yaw Change</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Markers</string>
           </property>
           <item>
            <property name="text">
             <string>Section Boundaries</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
           </item>
           <item>
            <property name="text">
             <string>POV Position</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="transitionWidget" name="transitionEditor">
       <attribute name="title">
        <string>Transition Editor</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QCustomPlot" name="plotter" native="true">
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <widget class="Line" name="povLine">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>0</y>
            <width>2</width>
            <height>301</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>4</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>transitionWidget</class>
   <extends>QWidget</extends>
   <header>transitionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
